zebra/zebra_vty.c

```diff
@@ -3654,10 +3654,17 @@ DEFUN_HIDDEN (zebra_packet_process,
 	      "Number of packets to process before relinquishing thread\n")
 {
 	uint32_t packets = strtoul(argv[2]->arg, NULL, 10);
+	struct listnode *node = NULL;
+	struct zserv *client = NULL;
 
 	atomic_store_explicit(&zrouter.packets_to_process, packets,
 			      memory_order_relaxed);
 
+	zlog_info("Setting the Zapi packet to process to %d", packets);
```

Comment:
Oh, I don't think we should log in the cli handler

Reply:
removed this